mmv1/products/vertexai/api.yaml

```diff
@@ -277,6 +277,92 @@ objects:
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: True
+        allowed:
+          - True
+          - False
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+      include_project: true
+    description: |-
+      Feature Metadata information that describes an attribute of an entity type. For example, apple is an entity type, and color is a feature that describes apple.
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: entitytype
+        description: The name of the Featurestore to use, in the format projects/{project}/locations/{location}/featurestores/{featurestore}/entityTypes/{entitytype}.
+        url_param_only: true
+        input: true
+        required: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: The name of the feature. The feature can be up to 64 characters long and can consist only of ASCII Latin letters A-Z and a-z, underscore(_), and ASCII digits 0-9 starting with a letter. The value will be unique given an entity type.
+        input: true
+        url_param_only: true
+        pattern: '{{entitytype}}/features/{{name}}'
+      - !ruby/object:Api::Type::String
+        name: 'etag'
+        description: Used to perform consistent read-modify-write updates.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          The timestamp of when the entity type was created in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::String
+        name: 'updateTime'
+        output: true
+        description: |
+           The timestamp when the entity type was most recently updated in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: |
+          A set of key/value label pairs to assign to the feature.
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: Description of the feature.
+      - !ruby/object:Api::Type::Enum
+        name: 'valueType'
+        description: |
+          Type of Feature value. Immutable.
+        required: true
+        values:
+          - VALUE_TYPE_UNSPECIFIED
+          - BOOL
+          - BOOL_ARRAY
+          - DOUBLE
+          - DOUBLE_ARRAY
+          - INT64
+          - INT64_ARRAY
+          - STRING
+          - STRING_ARRAY
+          - BYTES
```

Comment:
could this be transitioned to a string with a link to https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.featurestores.entityTypes.features#ValueType 

We find enums difficult to maintain

Reply:
Updated the code in 34219f2.